opm/input/eclipse/Schedule/Well/WListManager.hpp

```diff
@@ -62,6 +62,7 @@ class WListManager
     std::size_t getNoWListsWell(const std::string& wname) const;
     bool hasWList(const std::string& wname) const;
     void addWListWell(const std::string& wname, const std::string& wlname);
+    void addWListWells(const std::vector<std::string>& wname, const std::string& wlnames);
```

Comment:
There are different parameter orders in the functions here. I feel like it is easier if we have well list name first and well name(s) as the second argument.